docs/arcaflow/workflows/expressions.md

```diff
@@ -261,6 +261,7 @@ Functions:
 | `toUpper(string)`                                | string       | Returns the input with Unicode letters mapped to their upper case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `splitString(string, string)`                    | list[string] | Returns a list of the substrings which appear between instances of the specified separator; the separator instances are not included in the resulting list elements; adjacent occurrences of separator instances as well as instances appearing at the beginning or ending of the input will produce empty string list elements.<br>&nbsp; Param 1: The string to split.<br>&nbsp; Param 2: The separator.                                                                                     |
 | `readFile(string)`                               | string |  Returns the contents of a file as a UTF-8 character string, given a file path string. Relative file paths are resolved from the Arcaflow process working directory. Shell environment variables are not expanded.                                                                                     |
+| `bindConstants(list[any], any)`                               | list[object] |  Returns a list of objects with a new ID equal to schema name of the first parameter concatenated with the schema name of the second parameter. For each item in the list, an object with two properties is created. The item is assigned to this object's `item` property. The second argument of the `bindConstants` function is assigned to this object's `constant` property, thus repeating the second argument for each item in the list.                                                                                   |
 
```

Comment:
I would rephrase the description slightly:
> Returns a list of objects each containing two properties:   an `item` property which contains the corresponding item from the list in the first parameter; and, a `constant` property which contains the value of the second input parameter.  The ID of the return value schema is the concatenation of the schema name of the first parameter and the schema name of the second parameter.

The schema ID really needs a concrete example, although perhaps it would be better presented as part of a whole usage example elsewhere, as Jared suggested.